content/blog/doomsday-rule/index.md

```diff
@@ -0,0 +1,660 @@
+---
+{
+    title: "Doomsday Rule",
+    description: 'In this blog I talk about the Doomsday Rule, how it works, how to put it into code then how to make a program that tests you.',
+    published: '2019-06-29T22:12:03.284Z',
+    authors: ['SkyHawk_0'],
+    tags: ['Python', 'Maths', 'Doomsday rule'],
+    attached: [],
+    license: 'cc-by-4'
+}
+---
+
+# What is the doomsday rule?
+
+Before I get to how my program works, I should probably explain what the doomsday rule is. The doomsday rule is an algorithm that determines the day of the week for a given date in the Gregorian calendar, e.g. the 18th of March 1898 is a Friday. I can't think when you would use this in everyday life but it's a cool party trick to show off with.
+
+The algorithm was invented by John Conway in 1973. You may know some of his work like Conway's game of life. A nice little fact about the dates that Conway was born on and died on were doomsdays but I will tell what they are later. The algorithm is simple enough that it can be calculated mentally. Conway could do this in a couple of seconds, and to improve his speed he created a program which gave him ten random dates every time he logged on to his computer.
```

Comment:
```suggestion
The algorithm was invented by John Conway in 1973. You may know some of his work, like Conway's game of life. Here's a nice little fact: the dates that Conway was born and died on were both doomsdays. We'll explain what doomsdays are later. The algorithm is simple enough that it can be calculated mentally. Conway could do this in a couple of seconds, and to improve his speed, he created a program that gave him ten random dates every time he logged on to his computer.
```

Rewrote the "nice fact" sentence to read a little bit smoother. It was difficult for me to follow originally, but that **is** a fun fact!